<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,0">
   <item>
    <widget class="QTableWidget" name="twStudent">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="cboHHeader">
             <property name="text">
              <string>行表头</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cboVHeader">
             <property name="text">
              <string>列表头</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cboAltemate">
             <property name="text">
              <string>交替背景色</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cboEditCell">
             <property name="text">
              <string>单元格可编辑</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnStyleSheet">
           <property name="text">
            <string>设置样式表</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="rbCell">
           <property name="text">
            <string>选择单元格</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbRow">
           <property name="text">
            <string>选择行</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leName"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>籍贯：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leAge"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>性别：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leGender"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leProvince"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>年龄：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnAppend">
           <property name="text">
            <string>添加一行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnInsert">
           <property name="text">
            <string>插入一行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>删除当前行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnModify">
           <property name="text">
            <string>修改当前行</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
